src/configs/firebase.ts

```diff
@@ -0,0 +1,13 @@
+import { initializeApp } from 'firebase/app';
+import { getAuth } from 'firebase/auth';
+const firebaseConfig = {
+  apiKey: 'AIzaSyBBf9hUi5bVA0m9eMa79VzHiHE1dPWvBjg',
```

Comment:
It isn't a good idea to add info about apiKey in GIT.

Reply:
API key is visible in request URL anyway. 
It seems that we can hide in .env the rest of FB config.